src/main/java/seedu/duke/classes/Show.java

```diff
@@ -13,8 +13,20 @@
     protected int currentSeason;    //to keep track of watch progress
     protected int currentEpisode;
 
-    public Show() {
 
+    @Override
+    public String toString() {
+        String des = name + " | ";
+        des += "WatchHistory : S";
+        des += Integer.toString(currentSeason);
+        des += "E";
+        des += Integer.toString(currentEpisode);
+        if (rating != -1) {
+            //TODO : make sure a review is always passed in with a rating
+            des += "| Rating: ";
+            des += Integer.toString(rating);
+        }
+        return des;
```

Comment:
I think you should not put this inside the constructor

Reply:
No it's not inside the constructor, I'll move it to the bottom so it's clearer haha